index.html

```diff
@@ -60,8 +60,242 @@ <h1>Hey there. I’m Dhuruv <br><span class="h-green">I’m a software developer
+                        </li>
+                        <li>
+                            <p>css</p>
+                        </li>
+                        <li>
+                            <p>JavaScript</p>
+                        </li>
+                        <li>
+                            <p>html</p>
+                        </li>
+                    </ul>
+                    <div class="project-btn-sec">
+                        <button type="button">See Project</button>
+                    </div>
+                </div>
+            </div>
+            <div class="card-item">
+                <div class="text-box">
+                    <h4 class="project-title">Multi-Post Stories Gain+Glory</h4>
+                    <ul class="project-tag">
+                        <li>
+                            <p>Ruby on rails</p>
+                        </li>
+                        <li>
+                            <p>css</p>
+                        </li>
+                        <li>
+                            <p>JavaScript</p>
+                        </li>
+                        <li>
+                            <p>html</p>
+                        </li>
+                    </ul>
+                    <div class="project-btn-sec">
+                        <button type="button">See Project</button>
+                    </div>
+                </div>
+            </div>
+            <div class="card-item">
+                <div class="text-box">
+                    <h4 class="project-title">Multi-Post Stories Gain+Glory</h4>
+                    <ul class="project-tag">
+                        <li>
+                            <p>Ruby on rails</p>
+                        </li>
+                        <li>
+                            <p>css</p>
+                        </li>
+                        <li>
+                            <p>JavaScript</p>
+                        </li>
+                        <li>
+                            <p>html</p>
+                        </li>
+                    </ul>
+                    <div class="project-btn-sec">
+                        <button type="button">See Project</button>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="about-me">
+            <div class="about-me-text">
+                <h1>About me</h1>
+                <p> I can help you build a product , feature or website Look through some of my work and experience! If
+                    you like what you see and have a project you need coded,
+                    don’t hestiate to contact me.
+                </p>
+                <button type="button">Get My Resume</button>
+            </div>
+            <div class="about-me-illustration">
+                <img class="img-1" src="./icons/about-me-illustration.png" alt="about me illustraion">
+                <img class="img-2" src="./icons/about-me-illustration2.png" alt="about me illustraion">
```

Comment:
- [ ] Please use the same margins and padding as in Figma desing

Reply:
I checked everything looks fine margin and padding